lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -6,6 +6,8 @@ const path = require('path');
 
 module.exports = {
   update() {
+    const templateUrl = `https://s3.amazonaws.com/${this.bucketName}/compiled-cloudformation-template.json`;
```

Comment:
Thanks @nicka / @eahefnawy for your comments!
Yep. Was also thinking about to add a timestamp and deploy it (like we do with the 5 last versions of the .zip file) but IMHO it's too much overhead for now. Maybe we can include this in another PR but dealing with the S3 bucket cleanup is kind of tedious.